pages/components/Todolist.tsx

```diff
@@ -14,6 +14,17 @@ const Todolist = () => {
     const [isModalOpen, setIsModalOpen] = useState(false);
     const [todoToDelete, setTodoToDelete] = useState<number | null>(null);
 
+    useEffect(() => {
+        const storedTodos = localStorage.getItem('todos');
+        if (storedTodos) {
+            setTodos(JSON.parse(storedTodos));
+        }
+    }, []);
+
+    useEffect(() => {
+        localStorage.setItem('todos', JSON.stringify(todos));
+    }, [todos]);
```

Comment:
do we need useEffect here?
can we set this in the `addTodo` and `editTodo` func instead